### PR TITLE
support startpage and duckduckgo

### PR DESCRIPTION
**Pull request by ttttmr**
#20

### CODE REVIEW

**Comment by aboutmydreams** [comment]
cool, i think it's very valuable

File: src/content-script/index.mjs [hunk 1]
```diff
@@ -28,11 +21,40 @@ async function run(question) {
...
+    document.getElementById("rcnt").appendChild(container);
+  }
+  const searchInput = document.getElementsByName("q")[0];
+  if (searchInput && searchInput.value) {
+    // only run on first page
+    const startParam = new URL(location.href).searchParams.get("start") || "0";
+    if (startParam === "0") {
+      run(searchInput.value);
+    }
+  }
```
Comment (1zizi1) [inline src/content-script/index.mjs:58]: Maybe refactor this into a function so that other search engines can be cleanly added

**Review by 1zizi1 — CHANGES_REQUESTED** [review]

**Comment by Julian88Tex** [comment]
Partial duplicate of #8

**Comment by wong2** [comment]
https://github.com/wong2/chat-gpt-google-extension/pull/31

**Pull request closed without merging**